hermes/bin/src/cli/run.rs

```diff
@@ -61,6 +61,7 @@ impl Run {
             Emoji::new("🛠️", ""),
             app.name()
         );
+        // TODO[RC]: Prevent the app from receiving any events until it is fully initialized.
```

Comment:
It seems that the comment from test should (also) be here. 
```suggestion
        // TODO[RC]: Prevent the app from receiving any events until it is fully initialized.
        // TODO[RC]: Currently, when a module fails to initialize, the whole app fails to run.
```

Reply:
Applied suggestion